src/node/desktop/src/ui/widgets/choose-r/ui.html

```diff
@@ -25,6 +25,17 @@
 
     <select id="select" class="select" name="select" size="7"></select>
 
+    <p id="i18n-customizeRenderingEngine"></p>
+
+    <div>
+      <label id="i18n-renderingEngineLabel" for="rendering-engine"></label>
+      <select name="rendering-engine" id="rendering-engine">
+        <option label="Auto-detect (recommended)" value="auto"></option>
+        <option label="Desktop" value="desktop"></option>
+        <option label="Software" value="software"></option>
```

Comment:
Can we put these UI strings into the string file instead of hardcoding them? Ideally, new UI should be localizable from the start.